packages/rum/src/boot/rum.entry.spec.ts

```diff
@@ -1,11 +1,15 @@
-import { ErrorSource, ONE_SECOND } from '@datadog/browser-core'
+import { Configuration, ErrorSource, ONE_SECOND } from '@datadog/browser-core'
 import { setup, TestSetupBuilder } from '../../test/specHelper'
 import { ActionType } from '../domain/rumEventsCollection/action/trackActions'
 import { makeRumGlobal, RumGlobal, RumUserConfiguration, StartRum } from './rum.entry'
 
 const noopStartRum = () => ({
   addAction: () => undefined,
   addError: () => undefined,
+  addTiming: () => undefined,
```

Comment:
could we add some tests for it here?
it could have caught 740ffb1508c15b71c804ef80eb2c7b1409d64596

Reply:
Right, I will add a test for it.